src/models/location.model.js

```diff
@@ -0,0 +1,10 @@
+class LocationModel {
+  constructor({ city, country }) {
+    this.city = city;
+    this.country = country;
+  }
+  static create(m) {
+    return new LocationModel(m);
```

Comment:
```suggestion
  static create(data) {
    return new LocationModel(data);
```
Te rrecomiendo le des un nombre mas sugestivo a tus variables. De resto esta todo super